loky/initializers.py

```diff
@@ -0,0 +1,78 @@
+import warnings
+
+
+def _viztracer_init(init_kwargs):
+    """Initialize viztracer's profiler in worker processes"""
+    from viztracer import VizTracer
+    tracer = VizTracer(**init_kwargs)
+    tracer.register_exit()
+    tracer.start()
+
+
+def _make_viztracer_initializer_and_initargs():
+    try:
+        import viztracer
+        tracer = viztracer.get_tracer()
+        if tracer is not None and getattr(tracer, 'enable', False):
```

Comment:
@tomMoral this was the cause of the previous side-effect that caused a failure in a downstream test: the profiler would still be initialized in the in the crashing executor.

Reply:
Oh, ok! If it is stop, it does not disapear, it is just not enabled...
Make sense. Thx for the fix.